### DISCUSSION TITLE
How the site was created?

### DISCUSSION THREAD

**Discussion by gioele-ferrari** [cssbattle/forum#133] [Q&A]
Hey guys, does anyone know with which **framework** the site is made? I'm so curious about how it was made and if anyone (maybe the devs too) had any ideas, I'd be happy! (obviously I think there is basic html css and js).

**Comment by chinchang** [comment]
Its written in react on the frontend. CSS modules with BEM naming convetion for CSS.

**Reply by rafehimself** [reply]
and the back-end?
How are solutions calculated ? is that python machine learning?

**Reply by chinchang** [reply]
backend is on Firebase & cloud functions
